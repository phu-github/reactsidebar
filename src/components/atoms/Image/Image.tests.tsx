import React from 'react';
import renderer from 'react-test-renderer';
import {Image} from '.';
import DropdownImg from '../../../assets/images/ph2-logo.svg';

describe('Image component tests', () => {
  it('Should render correctly with default props', () => {
    const component = renderer.create(<Image srcImg={DropdownImg} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
