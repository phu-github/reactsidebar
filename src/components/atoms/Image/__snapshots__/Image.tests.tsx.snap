// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image component tests Should render correctly with default props 1`] = `
<img
  className="image-style"
  src={
    Object {
      "ReactComponent": [Function],
      "default": [Function],
    }
  }
  style={
    Object {
      "backgroundColor": "tranparent",
      "border": 0,
      "borderRadius": "0px",
      "height": "auto",
      "margin": "0px",
      "width": "200px",
    }
  }
/>
`;
